forem/forem
# Switch node-sass to dart-sass

## What type of PR is this? (check all applicable)


## Description

We explicitly use `node-sass` to compile Storybook, but [node-sass/lib-sass has been deprecated](https://sass-lang.com/blog/libsass-is-deprecated) and has been replaced by [dart-sass](https://github.com/sass/dart-sass).

This PR makes this change and silences some CI logs for StoryBook.

## Related Tickets & Documents

Webpacker is still dependent on node-sass, but it'll soon change when they release version 6:

- https://github.com/rails/webpacker/issues/2673
- https://github.com/rails/webpacker/issues/2673

## QA Instructions, Screenshots, Recordings

Run `yarn storybook-prerequisites` and `yarn build-storybook` locally or `yarn storybook` and see it all works

To check that some logs are suppressed run `CI=true yarn storybook-prerequisites`

## Added tests?

- [x] No, and this is why: just reorganizing CLI commands
